Lib/test/test_tokenize.py

```diff
@@ -1341,13 +1342,13 @@ def assertExactTypeEqual(self, opstr, *optypes):
         tokens = list(tokenize(BytesIO(opstr.encode('utf-8')).readline))
         num_optypes = len(optypes)
         self.assertEqual(len(tokens), 2 + num_optypes)
-        self.assertEqual(token.tok_name[tokens[0].exact_type],
-                         token.tok_name[ENCODING])
+        self.assertEqual(tokenize_tok_name[tokens[0].exact_type],
+                         tokenize_tok_name[ENCODING])
```

Comment:
I would prefer to import the tokenize module and write tokenize.tok_name.

I know that this file uses "from tokenize import ..." but this pattern is unusual in the Python test suite.